pkg/gitprovider/git_provider.go

```diff
@@ -123,11 +124,15 @@ func (a *AbstractGitProvider) parseSshGitUrl(gitURL string) (*StaticGitContext,
 	repo.Name = matches[3]
 	repo.Id = matches[3]
 
-	repo.Url = getCloneUrl(repo.Source, repo.Owner, repo.Name)
+	repo.Url = getCloneUrl(repo.Source, repo.Owner, repo.Name, true)
 
 	return repo, nil
 }
 
-func getCloneUrl(source, owner, repo string) string {
-	return fmt.Sprintf("https://%s/%s/%s.git", source, owner, repo)
+func getCloneUrl(source, owner, repo string, isHttps bool) string {
+	scheme := "https"
+	if !isHttps {
```

Comment:
Please change this to
```golang
scheme := "http"
if isHttps { ....}
```

It's generally best to avoid negating statements to improve readability.